invenio_app_rdm/config.py

```diff
@@ -615,3 +616,16 @@ def files_rest_permission_factory(obj, action):
     "invenio_app_rdm/records/details/side_bar/export.html",
 ]
 """Template names for detail view sidebar components"""
+
+
+
+
+
+RDM_SEARCH_USER_DASHBOARD_COMMUNITIES = {
```

Comment:
Less space above here? Maybe to be put close to the `RDM_SEARCH_DRAFTS` one?
Question: shall we change the var `RDM_SEARCH_DRAFTS` to be something like `RDM_SEARCH_USER_DASHBOARD_UPLOADS`? Warning: breaking change!

Reply:
It makes sense! I can change and note it down as a breaking change for our release notes! WDYT?

Reply:
Hmmm I checked a bit and `RDM_SEARCH_DRAFTS` is defined in [rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/blob/f5f23882695b87eb92dae44a8b2bdac7b2ccac5c/invenio_rdm_records/config.py#L417) which doesn't know about dashboard. My thought now is if we should leave it like this and maybe just use it for the list of user uploads even if the naming is not ideal? Or create the `RDM_SEARCH_USER_DASHBOARD_UPLOADS ` and by default assign it to `RDM_SEARCH_DRAFTS`? For communities and requests, it might make sense to keep 2 configs e.g `RDM_SEARCH_USER_DASHBOARD_COMMUNITIES`, `RDM_SEARCH_COMMUNITIES` as we have 2 views that we search for communities and we might need to differentiate the configuration(?).